### ISSUE TITLE
Build failure on Raspberry Pi

### ISSUE THREAD

**Issue by casm** [btolab/hypseus#3]
Having a difficult time getting Hypseus to build on a Raspberry Pi 3 Model B under gcc-4.9.2.  cmake apparently runs fine, but `make` gives the following build failure:

pi@retropie:~/build/hypseus/build $ make
Scanning dependencies of target libmpeg2
[  1%] Creating directories for 'libmpeg2'
[  2%] Performing download step (download, verify and extract) for 'libmpeg2'
-- downloading...
     src='https://github.com/btolab/libmpeg2/archive/master.zip'
     dst='/home/pi/build/hypseus/build/3rdparty/src/master.zip'
     timeout='none'

-- [download 100% complete]
-- downloading... done
-- verifying file...
     file='/home/pi/build/hypseus/build/3rdparty/src/master.zip'
-- verifying file... warning: did not verify file - no URL_HASH specified?
-- extracting...
     src='/home/pi/build/hypseus/build/3rdparty/src/master.zip'
     dst='/home/pi/build/hypseus/build/3rdparty/src/libmpeg2'
-- extracting... [tar xfz]
-- extracting... [analysis]
-- extracting... [rename]
-- extracting... [clean up]
-- extracting... done
[  3%] No patch step for 'libmpeg2'
[  4%] No update step for 'libmpeg2'
[  5%] Performing configure step for 'libmpeg2'
configure.ac:131: error: possibly undefined macro: AC_LIBTOOL_WIN32_DLL
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:132: error: possibly undefined macro: AC_PROG_LIBTOOL
autoreconf: /usr/bin/autoconf failed with exit status: 1
CMakeFiles/libmpeg2.dir/build.make:103: recipe for target '3rdparty/src/libmpeg2-stamp/libmpeg2-configure' failed
make[2]: **\* [3rdparty/src/libmpeg2-stamp/libmpeg2-configure] Error 1
CMakeFiles/Makefile2:103: recipe for target 'CMakeFiles/libmpeg2.dir/all' failed
make[1]: **\* [CMakeFiles/libmpeg2.dir/all] Error 2
Makefile:96: recipe for target 'all' failed
make: **\* [all] Error 2

Any ideas appreciated; I'm not sure why I'm seeing this.

**Comment by maksun** [comment]
Hello,
based on this error message :
autoreconf: /usr/bin/autoconf failed with exit status: 1
It seems dh-autoreconf package is missing, install it and give it another try ;)
Regards,

**Comment by ghost** [comment]
it does compile fine on the pi here

**Comment by OzFalcon** [comment]
I think your missing libtool
> apt-get install libtool

**Comment by Darknior** [comment]
It compile fine for me too.
But i only have sound and no image :(

**Comment by lmerckx** [comment]
@Darknior Have you tried the "-useoverlaysb 2" parameter ?

**Comment by Darknior** [comment]
@lmerckx no i don't know any parameters ?
Where to use it ? With make ? Or to launch hypseus ?

**Comment by lmerckx** [comment]
@Darknior When launching Hypseus

**Comment by Darknior** [comment]
Thanks a lot it works perfectly for me now <3
I also found this fork i will try, they fix some other issues ... 
https://github.com/grant2258/hypseus

**Comment by lmerckx** [comment]
Yep, some corrections from me.
Still in PR here.

**Comment by Darknior** [comment]
@lmerckx  Yes i see ... i really wish you will continue with others to improve this fork <3
I write about that [here](https://retropie.org.uk/forum/topic/18505/hypseus-new-daphne-fork-to-try-on-retropie/8) because i love this system and i dream to play all the games on pi one day ...
